webapi/src/test/scala/org/knora/webapi/e2e/E2ECookbookSpec.scala

```diff
@@ -0,0 +1,283 @@
+/*
```

Comment:
This looks like sample code copied and pasted from somewhere else. Does it really need to be in the Knora git repository?